### PR TITLE
Convert .rst to .md 

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Convert README.rst to README.md and HISTORY.rst to CHANGELOG.md

### CODE REVIEW

File: sdk/core/azure-mgmt-datalake-nspkg/setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     name='azure-mgmt-datalake-nspkg',
     version='3.0.1',
     description='Microsoft Azure Data Lake Management Namespace Package [Internal]',
-    long_description=open('README.rst', 'r').read(),
+    long_description=open('README.md', 'r').read(),
```
Comment (lmazuel) [inline sdk/core/azure-mgmt-datalake-nspkg/setup.py]: Need to add the long description type

File: sdk/core/azure-mgmt-nspkg/setup.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
     name='azure-mgmt-nspkg',
     version='3.0.2',
     description='Microsoft Azure Resource Management Namespace Package [Internal]',
-    long_description=open('README.rst', 'r').read(),
+    long_description=open('README.md', 'r').read(),
```
Comment (lmazuel) [inline sdk/core/azure-mgmt-nspkg/setup.py]: Need to add the long description type

File: sdk/core/azure-mgmt-datalake-nspkg/setup.py (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/core/azure-mgmt-datalake-nspkg/setup.py]: Could you explain what you mean? I see it used in a similar way [here](https://github.com/Azure/azure-sdk-for-python/blob/90fe8d9c0263ba739d6dad0bdd270707edbc141a/sdk/keyvault/azure-keyvault-nspkg/setup.py#L29)

Comment (lmazuel) [reply sdk/core/azure-mgmt-datalake-nspkg/setup.py]: you need
long_description_content_type="text/markdown",

File: sdk/batch/azure-batch/CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,6 @@
-.. :changelog:
+# Release History
 
-Release History
-===============
-
-8.0.0 (2019-8-5)
-++++++++++++++++++
+## 8.0.0 (2019-8-5)
```
Comment (mitchdenny) [inline sdk/batch/azure-batch/CHANGELOG.md]: I noticed that the date format between the changelog entries in these files are different. Some are YYYY-M-D and some are YYYY-MM-DD. Would be good to get them consistent whilst you are touching the files anyway.

Comment (chidozieononiwu) [reply sdk/batch/azure-batch/CHANGELOG.md]: thanks I didn't catch that. I think I'll leave that for library owners to correct. its a giant number of files and every moment this PR stay open means more merge conflicts occurring.

**Pull request merged**